src/pages/contact.js

```diff
@@ -1,18 +1,66 @@
-import {React} from "react";
- 
+import React, { useState } from "react";
+import '../contact.css';
+
 const Contact = () => {
+    const [expanded, setExpanded] = useState(null);
+
+    const toggleFAQ = (index) => {
+        setExpanded(expanded === index ? null : index);
+    };
+
     return (
-        <main>
-            <h1>CS (Data) Chapter</h1>
-            <p>
-                isc@datasektionen.se
-            </p>
-            <h1>Media Chapter</h1>
-            <p>
-                international@medieteknik.com
-            </p>
+        <main className="contact-page body">
+            <section className="chapter-contacts col-2">
+                <div className="contact-info">
+                    <h1>CS (Data) Chapter</h1>
+                    <p>isc@datasektionen.se</p>
+                </div>
+                <div className="contact-info">
+                    <h1>Media Chapter</h1>
+                    <p>international@medieteknik.com</p>
+                </div>
+            </section>
+
+            <section className="in-tem">
+                <h2>International Committee Members</h2>
+                <ul>
+                    <li><span className="name">Håvard Alstadheim</span> <span className="role">- President</span></li>
```

Comment:
Not President, But International Student Coordinator (ISC)
Håvard if ISC for the CS chapter, similarly Erik Sevelin and Elena Pan are ISC for the Media chapter.

Reply:
My bad, Im on it!